pymatviz/ptable.py

```diff
@@ -538,6 +536,43 @@ class ChildPlotters:
     (line/scatter/histogram) for individual element tiles.
     """
 
+    @staticmethod
+    def heatmap(
```

Comment:
Had to add another `heatmap` child plotter instead of reusing the `rectangle` plotter because `set_clip_path` doesn't seem to show the boarder for each tile (not sure if this is solvable). 

Another reason being I kind of want to integrate some of the processing steps for "percentage/ratio/..." inside the child plotter.

Let's see how everything goes and decide later.

Reply:
> because set_clip_path doesn't seem to show the border for each tile (not sure if this is solvable).

not sure, maybe something to ask an LLM 😄